docs/ABOUT.md

```diff
@@ -1,41 +1,18 @@
-
-Haskell is a purely functional, lazy, statically-typed programming
-language with type inference.
+Haskell is a purely functional, lazy, statically-typed programming language with type inference. 
+It has a beautiful and minmal syntax.
```

Comment:
Applicable if this sentence is chosen to remain:

spelling: minmal -> minimal

Reply:
I've updated the file to fix the typo